Alert QoL + Docs updates (#1111)
SleipnirGroup/Choreo: docs/choreolib/getting-started.md

        1. For more information about differential drive kinematics, see [WPILib's documentation](https://docs.wpilib.org/en/stable/docs/software/kinematics-and-odometry/differential-drive-kinematics.html). In this example, we assume you have created an instance of `DifferentialDriveKinematics`, named `kinematics`.

## Choreo-Specific Alerts

> Choreo primarily uses WPILib's Alerts api to warn users of error(or warning) conditions. These alerts

Reviewer: ```suggestion
Choreo primarily uses the WPILib Alerts API to provide users with internal warnings, errors or information. These alerts
```

> can be found under the SmartDashboard/ChoreoAlert section within networktables. 

Reviewer: Whys this line split

> To visualize these alerts on a dashboard or advantagescope, simply drag the ChoreoAlert group 

Reviewer: ```suggestion
To visualize these alerts in a dashboard such as AdvantageScope simply drag the ChoreoAlert group 
```

> outwards onto the "discrete fields" section in advantagescope or the main dashboard panel. 

Reviewer: whys this split?

Author: Oops ill fix it in format
